### PR TITLE
Remove duplicate conditions. Use switch instead.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: src/main/java/org/apache/commons/compress/harmony/pack200/MetadataBandGroup.java [hunk 1]
```diff
@@ -242,29 +242,48 @@ public void addParameterAnnotation(final int numParams, final int[] annoN, final
...
-            } else if (tag.equals("s")) {
-                final String value = (String) valuesIterator.next();
-                cases_RU.add(cpBands.getCPUtf8(value));
+            switch (tag) {
+                case "B":
+                case "C":
+                case "I":
+                case "S":
+                case "Z": {
+                    final Integer value = (Integer) valuesIterator.next();
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/compress/harmony/pack200/MetadataBandGroup.java:251]: in-line local vars?

File: src/main/java/org/apache/commons/compress/harmony/pack200/MetadataBandGroup.java [hunk 2]
```diff
@@ -242,29 +242,40 @@ public void addParameterAnnotation(final int numParams, final int[] annoN, final
...
-                casec_RS.add(cpBands.getCPSignature(value));
-            } else if (tag.equals("e")) {
-                final String value = (String) valuesIterator.next();
-                final String value2 = (String) valuesIterator.next();
-                caseet_RS.add(cpBands.getCPSignature(value));
-                caseec_RU.add(cpBands.getCPUtf8(value2));
-            } else if (tag.equals("s")) {
-                final String value = (String) valuesIterator.next();
-                cases_RU.add(cpBands.getCPUtf8(value));
+            switch (tag) {
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/compress/harmony/pack200/MetadataBandGroup.java:245]: Can `tag` be null here?

Comment (arturobernalg) [reply src/main/java/org/apache/commons/compress/harmony/pack200/MetadataBandGroup.java:245]: i guess

Comment (garydgregory) [reply src/main/java/org/apache/commons/compress/harmony/pack200/MetadataBandGroup.java:245]: @arturobernalg Would you please rebase on git master?

Comment (arturobernalg) [reply src/main/java/org/apache/commons/compress/harmony/pack200/MetadataBandGroup.java:239]: done

**Pull request merged**